aspects/schema.go

```diff
@@ -199,6 +199,10 @@ func (v *mapSchema) Validate(raw []byte) error {
 		return err
 	}
 
+	if mapValue == nil {
+		return fmt.Errorf(`cannot accept invalid null value for "map" type`)
```

Comment:
personally I would leave out "invalid", as it makes it sound like there could be a "valid null value"?  cc @degville